src/utils/helpers/pullErrors.js

```diff
@@ -10,6 +10,14 @@ const pullErrors = (errors) => {
   const email = [];
   const phone = [];
   const password = [];
+  const gender = [];
+  const birthday = [];
+  const preferredlanguage = [];
+  const currency = [];
+  const residentialaddress = [];
```

Comment:
Please check through this file to make sure all variable names are `camelCased` as that's the convention agreed on by the team